fastparquet/api.py

```diff
@@ -203,6 +204,34 @@ def filter_out_stats(rg, filters, helper):
     return False
 
 
+def statistics(obj):
+    if isinstance(obj, parquet_thrift.ColumnChunk):
+        md = obj.meta_data
+        s = obj.meta_data.statistics
+        rv = {}
+        if s.max is not None:
+            rv['max'] = encoding.read_plain(ensure_bytes(s.max), md.type, 1)[0]
+        if s.min is not None:
+            rv['min'] = encoding.read_plain(ensure_bytes(s.min), md.type, 1)[0]
+        if s.null_count is not None:
+            rv['null_count'] = s.null_count
+        if s.distinct_count is not None:
+            rv['distinct_count'] = s.distinct_count
+        return rv
```

Comment:
```
if se.converted_type:
    vmax = converted_types.convert(vmax, se)
```
where `se` is the corresponding schema-element, which is held only on the global file level (i.e., it is not referenced directly in the column chunk, row group, etc)